components/inputs/input-time.js

```diff
@@ -63,12 +65,19 @@ export function getTimeAtInterval(timeInterval, time) {
 	return time;
 }
 
-function initIntervals(size) {
-	if (!INTERVALS.has(size)) {
+function initIntervals(size, enforceTimeIntervals) {
```

Comment:
The render code was getting too messy with the end and start dropdown elements depending on `enforceTimeIntervals` so I moved it into here